Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -39,6 +39,7 @@ public class AirlockComponent : ServerDoorComponent, IWires
 
         private CancellationTokenSource _powerWiresPulsedTimerCancel = new CancellationTokenSource();
 
+        [ViewVariables]
```

Comment:
Those also have the properties already vv, so making the field vv too is pretty useless